src/study/system-model/include/antares/study/system-model/model.h

```diff
@@ -69,6 +70,7 @@ class Model
 
     const std::map<std::string, Variable>& Variables() const
     {
+        // TODO : convert to vector?
```

Comment:
why?

Reply:
because the current client code does not need the map structure; iterating through variables should be enough